.npmignore

```diff
@@ -0,0 +1,4 @@
+.github
+.eslintrc
+.eslintignore
+CHANGELOG.md
```

Comment:
the changelog absolutely belongs in the published package. offline access matters.